[Admin] (miscellaneous) adding descriptions to articles (#1668)
OfficeDev/office-js-docs-pr: docs/design/design-toolkits.md

---
title: Design toolkits for Office Add-ins

> description: 'Get links to some tools to help design the UI of an Office Add-in.'

Reviewer: ```suggestion
description: 'Learn about tools to help you design the user interface (UI) of an Office Add-in.'
```
